### ISSUE TITLE
Cache on Windows 10

### ISSUE THREAD

**Issue by eugene-beliaev** [jetli/create-yew-app#13]
I use Windows 10 and when I run "npm run start" I have to manually remove .cache folder after changes in code and then run "npm run start" again. Otherwise it always builds old changes, even if I changed some code and run "npm run start"

**Comment by jetli** [comment]
Hi, I don't have windows 10, but I cannot reproduce this on Mac OS, hot reloading is working on mac when editing codes.
Can you try again on Mac os or linux?

**Comment by eugene-beliaev** [comment]
I can of course try on Linux, but I'd like to use the tool on Windows as well, as it's my primary OS.

**Comment by Jupp56** [comment]
I can reproduce the problem as mentioned on Win10 and second on "I should be able to code on Windows".
Luckily, there is a workaround: WSL works perfectly. I had node and cargo already installed, so a quick ```cargo install wasm-pack``` later everything worked just fine, including the browser opening automatically.

If you want to do that: Install WSL (i use ubuntu), install node, install rust, install wasm-pack and you should be good to go.

**Comment by mirus-ua** [comment]
I can confirm. I have this issue on Windows 10 2004 19041.630
rustc 1.43.1 (8d69840ab 2020-05-04)
cargo 1.43.0 (2cbe9048e 2020-05-03)
node v12.16.3

**Comment by titaneric** [comment]
Same here, I find it little annoyed.

Windows 10 Home 20H2 19042.630
rustc 1.49.0-nightly (ffa2e7ae8 2020-10-24)
cargo 1.49.0-nightly (dd83ae55c 2020-10-20)
node v12.19.0
npm 6.14.8

**Comment by mirus-ua** [comment]
I investigated a bit and here what I got.
Here https://github.com/rustwasm/rust-parcel-template/issues/47 folks got the same issue.

HR not working only if files located inside the Windows file system. If you move them to WSL FS everything will work fine.

Here the question in which part of the bundle we got this issue?  

BTW a small offtopic: WSL2 didn't open the browser automatically with this error: `Unexpected error while opening in browser: true` someone knows how to fix it?
